### DISCUSSION TITLE
Suggestion: Display a clock

### DISCUSSION THREAD

**Discussion by Altersoundwork** [semaphoreui/semaphore#1407] [Ideas]
Just a simple suggestion. Since Semaphore has the option to specify via cron when to run a task, it'd be nice to have a simple clock on the semaphore GUI, anywhere really, so you're sure the time (and timezone) is correct.

**Comment by ansibleguy** [comment]
Just linking - related to: https://github.com/ansible-semaphore/semaphore/issues/1404
